From 4daa270c84825f41583e313b5b64bb10e6eae639 Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Fri, 28 Feb 2014 11:46:05 -0500
Subject: [PATCH] console: Fix unsetting 'resize-guest' property from UI

Reported in the followup of bz 754559

(cherry picked from commit 4a2493f1e4bede671eac713081f61965f036d6ed)
---
 virtManager/console.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/virtManager/console.py b/virtManager/console.py
index 6ac8abc..5199f5a 100644
--- a/virtManager/console.py
+++ b/virtManager/console.py
@@ -1092,7 +1092,7 @@ class vmmConsolePages(vmmGObjectUI):
 
     def resizeguest_ui_changed_cb(self, src):
         # Called from details.py
-        if not src.get_active():
+        if not src.get_sensitive():
             return
 
         val = int(self.widget("details-menu-view-resizeguest").get_active())
